### PR TITLE
Add sdn to ovn migration workflow and cluster profile for UPI AWS

### PR DESCRIPTION
**Pull request by weliang1**
https://issues.redhat.com/browse/OCPQE-9876:UPI AWS sdn migration to OVN

@anuragthehatter @zhaozhanqi @lihongan PTAL!

### CODE REVIEW

File: ci-operator/step-registry/cucushift/installer/rehearse/aws/upi/ovn/sdn2ovn/provision/cucushift-installer-rehearse-aws-upi-ovn-sdn2ovn-provision-chain.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+chain:
+  as: cucushift-installer-rehearse-aws-upi-ovn-sdn2ovn-provision
+  steps:
+  - chain: ipi-conf-aws
+  - chain: ipi-install
```
Comment (lihongan) [inline ci-operator/step-registry/cucushift/installer/rehearse/aws/upi/ovn/sdn2ovn/provision/cucushift-installer-rehearse-aws-upi-ovn-sdn2ovn-provision-chain.yaml:5]: Looks installer team is still working on the aws-upi base profile, we should wait until they complete the base profile. I think the steps above are for IPI install not UPI.

Comment (weliang1) [reply ci-operator/step-registry/cucushift/installer/rehearse/aws/upi/ovn/sdn2ovn/provision/cucushift-installer-rehearse-aws-upi-ovn-sdn2ovn-provision-chain.yaml:5]: @lihongan @jianlinliu Seems aws-upi base profile is still not available yet: https://github.com/openshift/release/tree/master/ci-operator/step-registry/cucushift/installer/rehearse/aw

Comment (jianlinliu) [reply ci-operator/step-registry/cucushift/installer/rehearse/aws/upi/ovn/sdn2ovn/provision/cucushift-installer-rehearse-aws-upi-ovn-sdn2ovn-provision-chain.yaml:5]: Yes, upi aws PR is pending on dev's review.

**Pull request closed without merging**